components/ui/useCode/Submitting.tsx

```diff
@@ -0,0 +1,11 @@
+import { Button } from "@/components/ui/button";
+import { Loader2 } from "lucide-react";
+
+export function FormSubmitting() {
+    return (
```

Comment:
You may inline this. It's easy enough to digest even being inlined.

Reply:
done